website/docs/advanced/instantiate_objects/overview.md

```diff
@@ -206,14 +206,68 @@ OmegaConf containers (DictConfig, ListConfig).
 OmegaConf containers have many advantages over primitive dicts and lists but in some cases 
 it's desired to pass a real dicts and lists (for example, for performance reasons).
 
-You can change the parameter conversion strategy using the `_convert_` parameter (in your config or the call-site).
+You can change the parameter conversion strategy using the `_convert_` parameter.
```

Comment:
Thanks! looks great overall, a couple of nits:
1. it'd be great to provide more context on why DictConfig would be preferred some time.
2. it'd be great to provide a bit more details on the performance comments, ie, which option could potentially speed up the instantiate call

Reply:
>     1. it'd be great to provide more context on why DictConfig would be preferred some time.

Done in https://github.com/facebookresearch/hydra/pull/2080/commits/dff94bf0a506e8d7184b86a72a6a951ab2bf5f1b.

>     2. it'd be great to provide a bit more details on the performance comments, ie, which option could potentially speed up the instantiate call

Done in https://github.com/facebookresearch/hydra/pull/2080/commits/f665b0759cefa1a8c11f5819f5b24d1af39f2f2f.